packages/wallets/src/components/Base/WalletDropdown/WalletDropdown.scss

```diff
@@ -95,7 +86,7 @@
         flex-direction: column;
         align-items: flex-start;
         z-index: 2;
-        margin-top: 0.4rem;
+        margin-top: -1.2rem;
```

Comment:
Try not to use negative margins. Instead use `margin-bottom`

Reply:
This margin-top is added to shift the dropdown item list higher and reduce the gap from the message-container, adding margin-bottom does not resolve this issue